src/crud/useWaveSurfer.tsx

```diff
@@ -87,27 +91,14 @@ export function useWaveSurfer(
   }, []);
 
   useEffect(() => {
-    console.log(
-      'new playing',
-      playing,
-      'ready?',
-      wsRef.current?.isReady,
-      'playing?',
-      wsRef.current?.isPlaying(),
-      'hasregion?',
-      regionRef.current !== undefined,
-      'looping?',
-      regionRef.current?.loop
-    );
     playingRef.current = playing;
     if (playingRef.current) {
       if (wsRef.current?.isReady) {
-        if (regionRef.current && regionRef.current.loop)
+        if (regionRef.current && regionRef.current.loop)//this still needs work
```

Comment:
We should probably use "TODO:" at the beginning of comments like this so we can easily search for them.